lib/spack/spack/main.py

```diff
@@ -358,6 +357,9 @@ def make_argument_parser(**kwargs):
         '--color', action='store', default='auto',
         choices=('always', 'never', 'auto'),
         help="when to colorize output (default: auto)")
+    parser.add_argument(
+        '-c', default=None, action="append", dest="config_vars",
```

Comment:
can you add `--config` as a long option here?

Reply:
Added :point_right: https://github.com/spack/spack/pull/22251/commits/14827b6b4da3a9f7d96575b137b5896c08815177